src/components/forms/DocumentUploadForm/UploadError.vue

```diff
@@ -1,13 +1,12 @@
 <template>
-  <div>
+  <div @click.prevent="onRemove">
```

Comment:
are you sure you need prevent default behavior? maybe you want to prevent bubbling instead or just artifact?